docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1209,49 +1206,181 @@ visual inspection and quantitatively using an appropriate statistical
+We
 might want to integrate this in the future, although I'm not sure if
 any PRL methods rely on the link between SSN and location.
 
-It is also possible that it will be annoying to Census Bureau if we
-have realistic SSN values, even if they are randomly generated, and we
-may wish to change to numeric format for this to a synthetic SSN-like
-(SSSN) value
+.. note::
 
+  In real life, people only get ITINs when they need them, i.e. when they file taxes for the first time.
+  In our simulation, we will initialize them right away and only observe them when the simulant files taxes,
+  which is essentially equivalent.
 
-.. sourcecode:: python
+Data sources and analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    # give everyone a unique fake ssn (for now)
-    df_population['ssn'] = [fake.unique.ssn() for _ in range(len(df_population))]
+The two non-trivial values are the SSN coverage among the foreign-born population
+of the US, and the SSN coverage among new foreign-born immigrants to the US.
 
-As a simple mechanism to capture some of the complexity in SSNs, we
-will have 10% of newborn simulants not receive a SSN.  We will also
-have 10% of simulants initialized at the beginning of the simulation
-not receive a SSN.
+For both calculations, we make the simplifying assumption that undocumented immigrants
+do not have SSNs, and documented immigrants do have SSNs.
+In reality, both parts of this are not quite right:
 
-**Verification and validation strategy**: to verify this approach, we
-can manually inspect a sample of 10-100 SSNs, confirm that the
-expected number are missing and that the duplication count follows the
-intended distribution.
+* Documented immigrants may not be authorized to work in the US.
+* Undocumented immigrants may have erroneously received an SSN, especially before
+  a reform to the process in 2001.
+
+For population initialization coverage, we use the ACS PUMS to estimate the
+foreign-born population of the United States,
+and a DHS report estimating the undocumented population in 2018. [DHS_Unauthorized]_
+
+For coverage at immigration, we use the ACS PUMS to estimate the
+foreign-born population who entered in the last year,
+and a demographic modeling analysis [Fazel-Zarandi_2018]_ estimating the yearly inflow of undocumented immigrants
+in 2017 (the most recent year reported).
+We assume that all undocumented immigrants are foreign-born.
+
+Simulation strategy
+^^^^^^^^^^^^^^^^^^^
+
+At all times in the simulation, all simulants have either an SSN **or** an ITIN.
+A simulant should never have both.
+
+SSN and ITIN should remain constant for a given simulant for their entire lifespan.
+
+We never switch someone who has an ITIN to have an SSN.
+
+.. note::
+
+  An SSN and an ITIN should not be treated like the same thing.
+  Logic in the taxes observer depends on which one a simulant has.
+
+SSN generation
+''''''''''''''
+
+**SSNs should be unique**.
+The same SSN should not be assigned to two different simulants.
```

Comment:
this is in conflict with some text up in the "project overview" section. non-urgent, but it should be updated to be in line with this